internal/reports/report_helpers.go

```diff
@@ -115,6 +117,14 @@ func buildTableReportBody(conv *internal.Conv, tableId string, issues map[string
 		}
 		sort.Strings(colNames)
 		var l []string
+		if p.severity == errors && len(tableLevelIssues) != 0 {
+			for _, issue := range tableLevelIssues {
```

Comment:
Will this generate a single issue combining all the issues on the table ?

Reply:
It is currently how its done for all the issues , I have just added handling for new table level issues. It appends all the issues into 1 table report body